### PR TITLE
Norweigan translation updates

### PR DESCRIPTION
**Pull request by AdamFx990**
I've been testing the changes uploaded by Grimdoc in issue #1373 for the past week and am happy for them to be merged.

### CODE REVIEW

**Comment by TheCycoONE** [comment]
@AdamFx990 please read the Travis output and correct whitespace problems.

**Comment by AdamFx990** [comment]
White space problems have been fixed. :)

**Comment by TheCycoONE** [comment]
@grimdoc are you happy with this pr

File: CorsixTH/Lua/languages/french.lua [hunk 1]
```diff
@@ -207,7 +207,7 @@ adviser.information = {
   patient_leaving_too_expensive = "Un patient part sans payer la facture pour %s. Sacrée perte !",
   vip_arrived = "Attention ! %s arrive pour visiter votre hôpital ! Faites en sorte de lui faire bonne impression.",
   first_death = "Vous venez de tuer votre premier patient. Alors, heureux ?",
-  promotion_to_specialist = "L'un de vos INTERNES est devenu MÉDECIN.", -- Fix the famous "Level 5 bug"
+  promotion_to_specialist = "L'un de vos MÉDECIN est devenu un SPÉCIALISTE.", -- Fix the famous "Level 5 bug"
```
Comment (wolfy1339) [inline CorsixTH/Lua/languages/french.lua]: This should be reverted. It was already fixed

**Comment by wolfy1339** [comment]
The first 3 commits are unrelated to this PR, and have already been merged in a different PR, they should be dropped.

**Comment by wolfy1339** [comment]
Are you able to do a rebase and remove those unrelated commits (first 3 and last 2)

**Comment by AdamFx990** [comment]
Sorry about the mess. I'm still learning git-bash.

**Comment by TheCycoONE** [comment]
Yeah, never make changes off your master branch - it makes a big mess.

```
git branch translations
git reset --hard origin/master
git checkout translations
git rebase -i master
(remove all lines not pertaining to this change, save and close)
git push origin --set-upstream translations
```
close this pr, and open a new one off that branch.

**Comment by TheCycoONE** [comment]
As an addendum, please read and follow:
https://github.com/CorsixTH/CorsixTH/blob/master/CONTRIBUTING.txt

**Comment by Grimdoc** [comment]
Sorry for the late reply, haven't been online for a while.
Yes, just add this :+1:

**Pull request closed without merging**